minecraft/mc262/src/main/cpp/vulkan_cuda_zerocopy.cpp

```
@@ -0,0 +1,255 @@
+        0,
+        static_cast<size_t>(g_width) * 4,
+        static_cast<size_t>(g_height),
+        cudaMemcpyDeviceToDevice
+    );
+    if (err != cudaSuccess) {
+        fprintf(
+            stderr,
+            "VulkanCudaZerocopy: cudaMemcpy2DFromArray failed: %s\n",
+            cudaGetErrorString(err)
+        );
+    }
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)